apps/aurora-portal/src/client/Auth/SignIn.test.tsx

```diff
@@ -1,5 +1,5 @@
 import { render, screen, fireEvent, waitFor } from "@testing-library/react"
-import { StoreProvider } from "../store/StoreProvider"
+import { GlobalStateProvider } from "../global-state/GlobalStateProvider"
```

Comment:
Global state for name I do not know :) lets just called it AuroraStore and provider AuroraStoreProvider, WDYT?